MultiTarget/MultiTargetToTargetFramework.props

```diff
@@ -1,4 +1,19 @@
-<Project>
+<Project InitialTargets="EnsureNoEnabledMultiTargetsWithMissingTfms">
+  <Target Name="EnsureNoEnabledMultiTargetsWithMissingTfms">
```

Comment:
Were these still showing up in the CI? If so, we can condition to debug config only?

Reply:
Sure, I can add that quick.